### DISCUSSION TITLE
Text overlap on mobile site

### DISCUSSION THREAD

**Discussion by aakashba** [alshedivat/al-folio#2124] [Q&A] [answered]
I'm seeing the publication year overlapping with the citation texts on mobile. Is there a way to avoid this? such as forcing an element to next line if another text element is in that position already? 

https://aakashba.github.io

**Comment by george-gca** [comment]
Our template was [thought to have](https://alshedivat.github.io/al-folio/publications/) either an image preview or the conference title in the left with the year on the right. Since you are not using that element, you'll probably want to do something more adaptative depending on the screen size. Check [bootstrap documentation about display](https://getbootstrap.com/docs/4.6/utilities/display/) and [what the sizes mean](https://getbootstrap.com/docs/4.6/layout/overview/#containers), and also how we used it in our template, for example in [_includes/repository/repo_trophies.liquid](https://github.com/alshedivat/al-folio/blob/master/_includes/repository/repo_trophies.liquid).

**Reply by aakashba** [reply]
Thanks! how do I add custom conference title/abbreviations? I can just do that to avoid this overlap!

**Reply by george-gca** [reply, marked as answer]
Add the field `abbr` on your [_bibliography/papers.bib](https://github.com/alshedivat/al-folio/blob/a7d6b37d8f82f31bf83e87afaf59b063abd08b30/_bibliography/papers.bib#L16C3-L16C14) file for each entry.
